monai/utils/enums.py

```diff
@@ -101,17 +112,6 @@ class BlendMode(Enum):
     GAUSSIAN = "gaussian"
 
 
-class PytorchPadMode(Enum):
-    """
```

Comment:
Looks like the changes in this file is not needed

Reply:
Sure, restored in the latest commit.
Thanks.